modules/cart/src/Plugin/Block/CartBlock.php

```diff
@@ -206,11 +205,25 @@ protected function getCartViews(array $carts) {
 
   /**
    * {@inheritdoc}
-   *
-   * @todo Find proper cache tags to make this cacheable
    */
-  public function getCacheMaxAge() {
-    return 0;
+  public function getCacheContexts() {
+    return Cache::mergeContexts(parent::getCacheContexts(), ['cart']);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheTags() {
```

Comment:
I thought this would just work with the cache tags from the cache context, but it does not. If you remove this bit the test fails. So even though the cache context's tags bust, these do not.

At the same time if the cache tags are not in the context it does not bust.